lib/utils/rename.js

```diff
@@ -1,16 +1,182 @@
 'use strict'
 var fs = require('graceful-fs')
+var path = require('path')
+var rimraf = require('rimraf')
```

Comment:
Unfortunately not.

gentlyRm takes paths relative to the npm prefix: https://github.com/npm/npm/blob/master/lib/utils/gently-rm.js#L54

Whereas `rename` takes paths relative to `process.pwd` (this is implicit in the call to `rename(2)`)

Changing over to using `gentlyRm` breaks tests; I'd need to refactor all callers of `rename` to make this change, and its not something I think should be in scope for this PR